nextmv/nextmv/cloud/run.py

```diff
@@ -164,7 +164,7 @@ class RunConfiguration(BaseModel):
     """Run type configuration for the run."""
     secrets_collection_id: Optional[str] = None
     """ID of the secrets collection to use for the run."""
-    queuing: Optional[RunQueueing] = None
+    queuing: Optional[RunQueuing] = None
     """Queueing configuration for the run."""
```

Comment:
```suggestion
    """Queuing configuration for the run."""
```

Here is another one. :blush:

Reply:
Thanks